docs/samples/Microsoft.ML.Samples/Dynamic/TextTransform.cs

```diff
@@ -31,12 +31,12 @@ public static void TextTransform()
 
             // Another pipeline, that customizes the advanced settings of the FeaturizeText transformer.
             string customizedColumnName = "CustomizedTextFeatures";
-            var customized_pipeline = ml.Transforms.Text.FeaturizeText(customizedColumnName, "SentimentText", s =>
-            {
-                s.KeepPunctuations = false;
-                s.KeepNumbers = false;
-                s.OutputTokens = true;
-                s.TextLanguage = TextFeaturizingEstimator.Language.English; // supports  English, French, German, Dutch, Italian, Spanish, Japanese
+            var customized_pipeline = ml.Transforms.Text.FeaturizeText(customizedColumnName, new List<string> { "SentimentText" }, 
```

Comment:
>new List<string> { "SentimentText" } [](start = 93, length = 36)

umm :)

Reply:
issue #2460 to discuss this

---
In reply to: [254357706](https://github.com/dotnet/machinelearning/pull/2394#discussion_r254357706) [](ancestors = 254357706)